espnet2/bin/enh_scoring_flexible_numspk.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
```

Comment:
This file has a lot overlap with espnet2/bin/enh_scoring.py. Can you add an option to inform flexible numspk in enh_scoring.py? Then just change the corresponding part.

Reply:
Added an option to enh_scoring.py